content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -88,16 +106,16 @@ Cypress will create a
 [`videosFolder`](/guides/references/configuration#Videos) to store the
 screenshots and videos taken during the testing of your application. Many users
 will opt to add these folders to their `.gitignore` file. Additionally, if you
-are storing sensitive environment variables in your configuration file
-(`cypress.json` by default) or
+are storing sensitive environment variables in your
+[Cypress configuration](/guides/references/configuration) or
 [`cypress.env.json`](/guides/guides/environment-variables#Option-2-cypress-env-json),
 these should also be ignored when you check into source control.
 
 </Alert>
 
 ### Test files
 
-Test files are located in `cypress/integration` by default, but can be
+Test files are located in `cypress/e2e` by default, but can be
 [configured](/guides/references/configuration#Folders-Files) to another
 directory. Test files may be written as:
```

Comment:
This needs to be updated to be to:
```suggestion
directory. Test files may be written as:

- `.js`
- `.jsx`
- `.ts`
- `.tsx`
```

Reply:
[17bd0f2](https://github.com/cypress-io/cypress-documentation/commit/17bd0f21aa3c3eb20cc204fc3d0fa27c950c06a5)